src/controllers/ContactController.ts

```diff
@@ -16,16 +16,21 @@ import { ContactFunction } from '../entity/enums/ContactFunction';
 @Tags('Contact')
 export class ContactController extends Controller {
   private async validateContactParams(req: express.Request) {
+    const emailOptionalFunctions = [
+      ContactFunction.SIGNATORY_AUTHORIZED,
+      ContactFunction.ASSISTING,
+    ];
     await validate([
       body('gender').isIn(Object.values(Gender)),
       body('firstName').trim(),
```

Comment:
Firstname is optional in the frontend if the user type is `Signatory_Authorized` or `Assisting`, so it should also be this way in the backend.